install_config/router/f5_router.adoc

```diff
@@ -363,4 +361,4 @@ $ oc adm router \
 The `external-host-username` is a *F5 BIG-IP* user account with access to the Advanced Shell (Bash) on the F5 BIG-IP system.
 ====
 
-The F5 setup is now ready, without the need to set up the ramp node.
+The F5 router plug-in is now ready, without the need to set up the ramp node.
```

Comment:
I'd remove this line.

Reply:
Assume that you are referring to line 364. If so I agree.